plenoptic/tools/signal.py

```diff
@@ -224,8 +253,11 @@ def rcosFn(width=1, position=0, values=(0, 1)):
 
 
 def pointOp(im, Y, X):
```

Comment:
we should consider changing the names of rcosFn and pointOp down the line because they are not intuitive/descriptive at all.

Reply:
indeed